docker/Dockerfile.nvidia.clang

```diff
@@ -0,0 +1,48 @@
+
+##############################################################
+# This Dockerfile contains NVIDIA HPC SDK (nvc, cuda, OpenMPI) and LLVM-15 for openmp offloading for Devito
+##############################################################
+FROM devito:nvidia
```

Comment:
Is it possible for a user to just pull nvidia.clang recipe ?

Reply:
The ideal would be to have

FROM devitocodes/devito/gpu-latest ?

Reply:
>Is it possible for a user to just pull nvidia.clang recipe ?

Well yeah that's how the devito image is built from the published push image from this build

>FROM devitocodes/devito/gpu-latest ?
No idea what you mean that's not how images are named.